src/format.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  camelCase: function camelCase (str) {
+    return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, (letter, index) => {
+      if (+letter === 0) return ``
+      return index === 0 ? letter.toLowerCase() : letter.toUpperCase()
+    })
+  },
+
+  kebabCase: function kebabCase (str) {
+    return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, (letter, index) => {
+      if (+letter === 0) return ``
+      return index === 0 ? letter.toLowerCase() : `-` + letter.toLowerCase()
+    })
+  },
+}
```

Comment:
That's no problem. I'm just pretty big on making sure that private modules do a single thing.